zooniverse/front-end-monorepo
# app-root: Add login param as prop to lib-user UserStats and MyGroups components

## Package
- app-root
- lib-user

## Describe your changes
- add `login` param in app-root as prop to lib-user components UserStats and MyGroups, adding functionality for an admin user to be able to access any user's user stats page or their "my groups" pages that lists that user's groups

## How to Review
_Helpful explanations that will make your reviewer happy:_
- What Zooniverse project should my reviewer use to review UX? n/a
- What user actions should my reviewer step through to review this PR?
  - logged in as an admin user:
    - load my user stats page: 
      - lib-user (yarn dev): https://local.zooniverse.org:8080/?users=markb-panoptes/stats
      - app-root (yarn start): https://local.zooniverse.org:3000/users/markbouslog/stats
    - load my groups (listing) page:
      - lib-user (yarn dev): https://local.zooniverse.org:8080/?users=markb-panoptes/groups
      - app-root (yarn start): https://local.zooniverse.org:3000/users/markbouslog/groups
- Which storybook stories should be reviewed? n/a
- Are there plans for follow up PR’s to further fix this bug or develop this feature? Yes, see linked GitHub project, and notably:
    - add containers to main components for cleaner code, can create stories and tests for components
    - pass adminMode boolean from app-root to lib-user, update usePanoptesUser and usePanoptesUserGroup hooks accordingly

# Checklist
_PR Creator - Please cater the checklist to fit the review needed for your code changes._
_PR Reviewer - Use the checklist during your review. Each point should be checkmarked or discussed before PR approval._

## General

## General UX
Example Staging Project: [i-fancy-cats](https://local.zooniverse.org:3000/projects/brooke/i-fancy-cats)
  - Can be used with a screen reader [BBC guide to VoiceOver](https://bbc.github.io/accessibility-news-and-you/assistive-technology/testing-steps/voiceover-mac.html)
  - Can be used from the keyboard [WebAIM guide to keyboard testing](https://webaim.org/techniques/keyboard/#testing)
  - It is passing accessibility checks in the storybook

## New Feature
  - Example of SlideTutorial [component](https://github.com/zooniverse/front-end-monorepo/blob/master/packages/lib-classifier/src/components/Classifier/components/SlideTutorial/SlideTutorial.js) with docgen comments, and its [story](https://zooniverse.github.io/front-end-monorepo/@zooniverse/classifier/index.html?path=/docs/other-slidetutorial--default)
